src/main/java/io/debezium/connector/yugabytedb/YugabyteDbConsistentStreaming.java

```diff
@@ -279,6 +279,11 @@ beginCountForTablet, message.tablet, new YBPartition(message.tablet),
         }
     }
 
+    @Override
+    public void commitOffset(Map<String, ?> offset) {
```

Comment:
Is the method required as this function does not have different behaviour to the base class?

Reply:
You're right, I have removed the method.